### PR TITLE
[1LP][RFR] Use common view class for timelines view

### PR DESCRIPTION
**Pull request by john-dupuy**
Fixing the `is_displayed` properties in 5.11. I would also like to fix cluster and host views since those are also impacting these tests. 

{{ pytest: --use-provider vsphere6-nested cfme/tests/infrastructure/test_timelines.py::test_infra_timeline_create_event }}

### CODE REVIEW

File: widgetastic_manageiq/__init__.py [hunk 1]
```diff
@@ -3106,12 +3106,12 @@ class chart(TimelinesChart):  # NOQA
...
     @property
     def is_displayed(self):
-        return self.title.text == "Timelines"
+        return self.title.text == "Timeline View"
 
     @property
     def is_timelines(self):
         """method to check title text for base Timelines without overriding is_displayed"""
-        return self.title.text == "Timelines"
+        return self.title.text == "Timeline View"
```
Comment (mshriver) [inline widgetastic_manageiq/__init__.py]: same change backported to 5.10? if it changed in specific z-stream I'd use VersionPicker again.

Comment (john-dupuy) [reply widgetastic_manageiq/__init__.py]: Yep, I believe that this is was just wrong and the `is_displayed` logic that uses this check got messed up as well.

Comment (john-dupuy) [reply widgetastic_manageiq/__init__.py]: Actually this is due to BZ https://bugzilla.redhat.com/show_bug.cgi?id=1732517, I modified this method to workaround that BZ

File: cfme/common/__init__.py [hunk 2]
```diff
@@ -818,3 +821,33 @@ def step(self, *args, **kwargs):
             table.click_at("Custom Button Events")
         else:
             raise DestinationNotFound("Custom button event count 0")
+
+
+class TimelinesView(View):
```
Comment (digitronik) [inline cfme/common/__init__.py]: It looks similar to `ServerTimelinesView` better to inherit and overwrite `is_displayed` method.

File: cfme/cloud/instance/__init__.py [hunk 3]
```diff
@@ -170,7 +170,20 @@ def is_displayed(self):
 
 
 class InstanceTimelinesView(TimelinesView, CloudInstanceView):
-    pass
+
+    @property
+    def is_displayed(self):
+        try:
+            check_object = self.breadcrumb.locations
+        except NoSuchElementException:
```
[the remark this answers is not in the document]
Comment (john-dupuy) [reply cfme/cloud/instance/__init__.py]: The problem is that in 5.10, there is no `breadcrumb` for the instance and infra VM timelines page. It is there on e.g. the provider's, host's and cluster's page but not this one. So this page needs to be specific to VM and instance.

File: cfme/common/__init__.py (hunk 2 above)
Comment (john-dupuy) [reply cfme/common/__init__.py]: Agreed, I can make this change.

File: cfme/cloud/instance/__init__.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (john-dupuy) [reply cfme/cloud/instance/__init__.py]: I tried using `VersionPicker` here, but it still tries to resolve the breadcrumb, which results in an `NoSuchElementException` error anyway, so I thought it best to simplify it to a `try/except` block.

**Pull request merged**